.travis.yml

```diff
@@ -2,5 +2,10 @@ language: ruby
 
 sudo: false
 
+before_install:
+  - wget -O toxiproxy-1.0.0.deb https://github.com/Shopify/toxiproxy/releases/download/v1.0.0/toxiproxy_1.0.0_amd64.deb
+  - sudo dpkg -i toxiproxy-1.0.0.deb
+  - sudo service start toxiproxy
```

Comment:
Provide a `script/install-toxiproxy-debian` for this so third-party developers can set this up as well, and include instructions for getting started in the README (run `script/setup` which runs `script/install-toxiproxy-debian`).